src/helpers.test.js

```diff
@@ -96,6 +97,43 @@ describe('getCurrentFilePath', () => {
   });
 });
 
+describe('getLocalPrettierPath', () => {
+  test('is null if no prettier package can be found', () => {
+    atomLinter.findCached.mockImplementation(() => undefined);
+    const filePath = path.join(__dirname, 'sourceFile.js');
+
+    const actual = getLocalPrettierPath(filePath);
+    const expected = undefined;
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('is prettier instance if one could be found', () => {
+    const prettierLib = path.join(__dirname, '..', 'node_modules', 'prettier', 'index.js');
+
+    atomLinter.findCached.mockImplementation(() => prettierLib);
+    const filePath = path.join(__dirname, '..', 'tests', 'fixtures', 'sourceFile.js');
+
+    const actual = getLocalPrettierPath(filePath);
+    const expected = prettierLib;
+
+    expect(actual).toBe(expected);
+  });
+
+  test('looks for prettier entry file', () => {
+    const prettierLib = path.join(__dirname, '..', 'node_modules', 'prettier', 'index.js');
+
+    atomLinter.findCached.mockImplementation(() => prettierLib);
+    const filePath = path.join(__dirname, '..', 'tests', 'fixtures', 'sourceFile.js');
+
+    getLocalPrettierPath(filePath);
+    const expectedDir = path.join(__dirname, '..', 'tests', 'fixtures');
+    const expectedLib = path.join('node_modules', 'prettier', 'index.js');
+
+    expect(atomLinter.findCached).toHaveBeenCalledWith(expectedDir, expectedLib);
+  });
```

Comment:
One is checking we get the right thing back given `findCached` returns it, the other checks we call `findCached` with the right arguments.

I had them in one test at first, but this way they can fail independently. Happy to merge them together if you'd prefer.